webpack.config.js

```diff
@@ -110,8 +117,17 @@ var config = {
 
 
         ],
+
+        preLoaders: [
+            // All output '.js' files will have any sourcemaps re-processed by 'source-map-loader'.
+            {
+                test: /\.js$/,
+                loader: "source-map-loader"
+            }
+        ]
     },
     'postcss': [require('autoprefixer')],
+    'devtool': 'source-map',
```

Comment:
The devtool property is actually being overwritten down below on env specific basis.  As long as it's in dev, i'm fine with any sourcemap option that is relatively performant (on build).  i had chosen inline-sourcemap.  But not with any great reason.